src/scenario-context/ScenarioContext.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+
+import { getBaselineScenario } from "../database";
+import { Scenario } from "../page-multi-facility/types";
+
+export type ScenarioUpdate = {
+  loading?: boolean;
+  failed?: boolean;
+  data?: Scenario | null;
+};
+
+interface ScenarioState {
+  loading: boolean;
+  failed: boolean;
+  data: Scenario | null;
+}
+
+type Action = { type: "update"; payload: ScenarioUpdate };
+type Dispatch = (action: Action) => void;
+
+const StateContext = React.createContext<ScenarioState | undefined>(undefined);
+const DispatchContext = React.createContext<Dispatch | undefined>(undefined);
+
+function scenarioReducer(state: ScenarioState, action: Action): ScenarioState {
+  switch (action.type) {
+    case "update":
+      return Object.assign({}, state, action.payload);
+  }
+}
+
+export const ScenarioProvider: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => {
+  const [state, dispatch] = React.useReducer(scenarioReducer, {
+    loading: true,
+    failed: false,
+    data: null,
+  });
+
+  useEffect(() => {
+    async function fetchBaselineScenario() {
+      const baselineScenario = await getBaselineScenario();
```

Comment:
We default to the baseline on the initial app load which is the desired functionality.  This should now be the only reference to baseline outside of the database code and the `CreateBaselineScenarioPage` component